client/engine/chainservice/eth_chainservice.go

```diff
@@ -21,16 +21,25 @@ type eventSource interface {
 	SubscribeFilterLogs(ctx context.Context, query ethereum.FilterQuery, ch chan<- ethTypes.Log) (ethereum.Subscription, error)
 }
 
+type TransactionWithStatusChan struct {
+	protocols.ChainTransaction
+	done chan struct{}
```

Comment:
Should the `done` channel emit a boolean? Or is the thought that the `done` channel eventually emits details about the submitted transaction?

Reply:
I'm not sure. What do you think would make the most sense?

Reply:
Maybe we should use a boolean for now as that is the simplest primitive for the current needs. We can change the interface later as required.

Reply:
OK. I think the actual simplest thing is the empty struct that is there now ⬆️ . This way, the chain service is signalling that it tried to send the transaction, and nothing more (no success / failure information or transaction hash or anything).